README.md

```diff
@@ -59,14 +63,14 @@ Umbrel integration is [work in progress](https://github.com/getumbrel/umbrel/pul
 
 Citadel integration is [work in progress](https://github.com/runcitadel/apps/pull/9).
 
-## ✨ Features
+## 🍊 Features
 
 - [x] Spending from the wallet
 - [x] Spending from the wallet via collaborative transactions
 - [x] Running the yield generator
-- [ ] TBD
+- [ ] [TBD](https://t.me/JoinMarketWebUI)
```

Comment:
fwiw, the telegram link should probably be changed too..?

Reply:
Yeah good point! cc @dergigi can you rename the group?